torch/csrc/distributed/rpc/rref_impl.h

```diff
@@ -207,6 +207,8 @@ class RRef : public RRefInterface {
     return rrefId_;
   }
 
+  virtual void tryDel() {}
```

Comment:
ideally, this should only live in `UserRRef`. This is OK for now, as otherwise we would have to do static_cast. After #30630, we should be able to let `confirmedUsers_` store `UserRRef`, and can then delete this from the base class.

Reply:
This was my concern too. Thanks for telling me about #30630.